src/views/Header.js

```diff
@@ -149,6 +157,7 @@ class Header extends React.PureComponent<void, HeaderProps, HeaderState> {
       return null;
     }
     const tintColor = this._getHeaderTintColor(props.navigation);
+    const truncatedBackTitle = this._getHeaderTruncatedBackTitle(props.navigation);
```

Comment:
`truncatedBackTitle` should be defined on the screen that also defines `backTitle`.